api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -137,10 +146,16 @@ private void checkResourceTypesAndOutputFormat(String resourceTypes, String outp
             log.error("Received invalid resourceTypes of {}", resourceTypes);
             throw new InvalidUserInputException("_type must be " + EOB);
         }
+
         if (outputFormat != null && !ALLOWABLE_OUTPUT_FORMAT_SET.contains(outputFormat)) {
             log.error("Received _outputFormat {}, which is not valid", outputFormat);
             throw new InvalidUserInputException("An _outputFormat of " + outputFormat + " is not valid");
         }
+
+        final boolean zipSupportOn = propertiesService.isToggleOn(ZIP_SUPPORT_ON);
+        if (!zipSupportOn && ZIPFORMAT.equalsIgnoreCase(outputFormat)) {
+            throw new InvalidUserInputException("An _outputFormat of " + outputFormat + " is not valid");
```

Comment:
This duplicates line 152 earlier. Should we at least extract the error message into a variable, or merge the two `if`s together with an `||`?

Reply:
I'll extract the error message into a common variable momentarily.

Reply:
done